R/create_query_data.R

```diff
@@ -581,7 +581,7 @@ assert_db_requirements <- function(version, version_arg_name, fun, fun_arg_name,
 #' # create a query for an SMQ
 #' library(tibble)
 #' library(magrittr, warn.conflicts = FALSE)
```

Comment:
I'm confused is the pipe used in this example??  why are we loading it if it is not being called?

Reply:
The pipe is used in line 609. However, as we load dplyr we do not need to load magrittr because dplryr reexports the pipe.

Reply:
This actually raises a good point, prior to this I believe a handful of the `library(dplyr, warn.conflicts = FALSE)` calls were for the pipe and a redundancy for `vars()`. When initially removing `vars()` it became imperative to include this library call, but now that we are bringing it back and _including_ the pipe operator, should I remove the unnecessary calls on the library?

Reply:
Yes, I would remove the unnecessary library calls.